crates/client/assets/schema.sdl

```diff
@@ -2,6 +2,12 @@ scalar Address
 
 scalar AssetId
 
+type AssetInfoDetails {
+	contractId: HexString!
```

Comment:
Not sure here, but maybe we want to use `ContractId!` here, as in [other places](https://github.com/FuelLabs/fuel-core/blob/master/crates/client/assets/schema.sdl#L594).